src/components/Sum.jsx

```diff
@@ -1 +1,14 @@
-// export function Sum() {}
+import React from 'react';
+
+export const Sum = ({ a, b }) => (
+  <>
+    <p>
+      {`Sum of ${a} and ${b} is ${a + b}`}
+    </p>
+  </>
```

Comment:
The closing fragment '</>' can be removed, as it's not necessary when there is only one child element.